test/s1/appframework/appframework_test.go

```diff
@@ -1091,6 +1126,9 @@ var _ = Describe("s1appfw test", func() {
 			// Wait for Standalone to be in READY status
 			testenv.StandaloneReady(deployment, deployment.GetName(), standalone, testenvInstance)
 
+			// Verify Monitoring Console is Ready and stays in ready state
```

Comment:
MC is not added to standalone. Standalone spec will need to be updated to contain MCREF.